### DISCUSSION TITLE
Remove `important` for specific styles

### DISCUSSION THREAD

**Discussion by LisaHao** [tailwindlabs/tailwindcss#14710] [Help]
Hi! I'm hoping to remove `!important` in our styles specifically just for our usages of `tracking-*`. We've found that our usage of `tracking-*` styles makes it so that users cannot override the letter-spacing if they need/want to do so for accessibility purposes.

Is there a way to still keep `important: true` at the configuration level but remove it just for this style type?

**Comment by wongjn** [comment]
If you are using Tailwind in a PostCSS build system, you could consider having a PostCSS plugin run after Tailwind that removes the `!important` flags as needed.
